west.yml

```diff
@@ -114,7 +114,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 98dc97fe88e22604c165a17ca81e7fbd5d69f82b
+      revision: f0a88d48e4617607bbb9061adac7d5748119ebaa
```

Comment:
@umapraseeda can you please include the changes from #5000 in this PR and point revision to https://github.com/nrfconnect/sdk-nrfxlib/commit/1f3ad531bf48d4241b6c60f5c521364a0785bfa7, and close #5000?

Reply:
Done @jfischer-no